### PR TITLE
FORMS-9757 avoid re-declaration of widget classes in case of loading …

### PR DESCRIPTION
**Pull request by barshat7**
…different form

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Review by rismehta — CHANGES_REQUESTED** [review]
Please add cypress test cases

File: it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+           function loadform(btn) {
+             var formPath = btn.getAttribute('data-form-path')
+             loadAdaptiveForm({
+               formPath: formPath,
+               CSS_Selector: '.customafsection'
+             })
+           }
+
+           var loadAdaptiveForm = function(options) {
+             var path = options.formPath + '.html?wcmmode=disabled';
```
Comment (rismehta) [inline it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html:13]: Can you write a comment that on publish instance the query parameters are not required

File: it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+             fetch(path).then((res) => res.text()).then(html => {
+               setInnerHTML(document.querySelector(options.CSS_Selector), html); // OR $(options.CSS_Selector).html(html);
+             })
+           }
+
+            function setInnerHTML(elm, html) {
+                elm.innerHTML = html;
+                Array.from(elm.querySelectorAll("script"))
```
Comment (rismehta) [inline it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html:21]: Why do you need to set script tag individually ? innerHTML should do it

File: it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html (hunk 1 above)
Comment (rismehta) [inline it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html:13]: how will this work if one needs to only show the form container (ie) without header/footer. In those case, we would have to also include the runtime library in the main page. Can you write a test case for this and also update the documentation ?

**Review by rismehta — CHANGES_REQUESTED** [review]
Check comments

File: it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html (hunk 2 above)
Comment (barshat7) [reply it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html:21]: script set through innerHTML does not execute automatically.
https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#:~:text=in%20this%20case-,Although%20this,-may%20look%20like
We can either use jquery or if jquery is not present then the given method is one of the ways to include scripts in embedded html.

**Pull request merged**